### DISCUSSION TITLE
Can you use Lambda Layers with Next Functions?

### DISCUSSION THREAD

**Discussion by thisispaul** [vercel/next.js#17522] [Help]
I can't find any documentation about this so it's probably not a feature but I wondered if it was possible to use a lamdba layer with a serverless function in next js?

For example if I have an api function where I was to use puppeteer package the headless browser in the function is far too big for a lambda function so it would be good to use something like a [chrome aws layer](https://github.com/shelfio/chrome-aws-lambda-layer).

**Comment by ChrisLusted** [comment]
@thisispaul  did you ever get something like this working? having issues fittign puppeter in a next api route myself

**Reply by ezalorsara** [reply]
Have you found the answer

**Reply by ChrisLusted** [reply]
I did, it's not really ideal but it worked. check out the issue here

https://github.com/vercel/vercel/issues/4739
